Quantum-Control-Applications/Superconducting/Single-Flux-Tunable-Transmon/09a_IQ_blobs.py

```diff
@@ -119,18 +120,16 @@
     waveform_report.create_plot(samples, plot=True, save_path=str(Path(__file__).resolve()))
 else:
     # Open the quantum machine
-    qm = qmm.open_qm(config)
+    qm = qmm.open_qm(full_config,close_other_machines=True)
     # Send the QUA program to the OPX, which compiles and executes it
     job = qm.execute(IQ_blobs)
     # Creates a result handle to fetch data from the OPX
     res_handles = job.result_handles
+    data_list = ["I_g", "Q_g", "I_e", "Q_e"]
     # Waits (blocks the Python console) until all results have been acquired
-    res_handles.wait_for_all_values()
+    results = res_handles.fetch_results(wait_until_done=False, timeout=60)
```

Comment:
Based on the commands and the script, I'd say that wait_until_done should be True, no?